visualization/app/codeCharta/ui/colorSettingsPanel/colorSettingsPanel.component.scss

```diff
@@ -18,6 +18,7 @@ color-settings-panel-component {
 	range-slider-component {
 		margin-top: 4px;
 		margin-left: 3px;
+		margin-bottom: -10px;
```

Comment:
Is there a way here to avoid negative margins? Those make structuring the website a lot harder in the future when they start affecting other components